{"uid":"b5d4c1757b82ac20","name":"test_upgrade_from_edge","fullName":"tests.integration.ha_tests.test_upgrade#test_upgrade_from_edge","historyId":"fb5755da30904890c6dc32bb58026031","time":{"start":1729302023475,"stop":1729302088032,"duration":64557},"status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/2","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/tests/integration/ha_tests/test_upgrade.py\", line 99, in test_upgrade_from_edge\n    await ops_test.model.wait_for_idle(\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/2","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1729301491048,"stop":1729301491049,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1729301491049,"stop":1729301491049,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1729301491049,"stop":1729301491255,"duration":206},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1729302023474,"stop":1729302023474,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1729302023474,"stop":1729302023475,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/2","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/tests/integration/ha_tests/test_upgrade.py\", line 99, in test_upgrade_from_edge\n    await ops_test.model.wait_for_idle(\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/2","steps":[],"attachments":[{"uid":"49eaaa36160e789a","name":"log","source":"49eaaa36160e789a.txt","type":"text/plain","size":1164}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"continuous_writes::finalizer","time":{"start":1729302088041,"stop":1729302094086,"duration":6045},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1729302094087,"stop":1729302094087,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1729302094091,"stop":1729302094477,"duration":386},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1729302094478,"stop":1729302094478,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1729302094478,"stop":1729302094478,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1729302094478,"stop":1729302094478,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_upgrade"},{"name":"host","value":"runner"},{"name":"thread","value":"17631-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_upgrade"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"48b57fe3a71d552d","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":4,"skipped":2,"passed":31,"unknown":1,"total":38},"items":[{"uid":"968fb5daafb0874b","reportUrl":"../114//#testresult/968fb5daafb0874b","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/2","time":{"start":1729215652762,"stop":1729215723813,"duration":71051}},{"uid":"111369eb9720ef61","reportUrl":"../113//#testresult/111369eb9720ef61","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/1","time":{"start":1729129290646,"stop":1729129364305,"duration":73659}},{"uid":"700f1399a5067567","reportUrl":"../112//#testresult/700f1399a5067567","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/2","time":{"start":1729042972674,"stop":1729043040428,"duration":67754}},{"uid":"2c659ded85ec642d","reportUrl":"../111//#testresult/2c659ded85ec642d","status":"passed","time":{"start":1728956424260,"stop":1728956722444,"duration":298184}},{"uid":"95906f5eaa92b542","reportUrl":"../110//#testresult/95906f5eaa92b542","status":"passed","time":{"start":1728870242113,"stop":1728870532734,"duration":290621}},{"uid":"8c87f2cb7c190d2a","reportUrl":"../109//#testresult/8c87f2cb7c190d2a","status":"passed","time":{"start":1728783615701,"stop":1728783925796,"duration":310095}},{"uid":"3bd4bddb1ee3ce43","reportUrl":"../108//#testresult/3bd4bddb1ee3ce43","status":"passed","time":{"start":1728697236112,"stop":1728697588703,"duration":352591}},{"uid":"c9d4de7202f9ec6b","reportUrl":"../107//#testresult/c9d4de7202f9ec6b","status":"passed","time":{"start":1728610789836,"stop":1728611097481,"duration":307645}},{"uid":"791f07bc02a30eaf","reportUrl":"../105//#testresult/791f07bc02a30eaf","status":"passed","time":{"start":1728438147301,"stop":1728438446571,"duration":299270}},{"uid":"c687157d27a01087","reportUrl":"../104//#testresult/c687157d27a01087","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1728351200793,"stop":1728351200793,"duration":0}},{"uid":"36b3d7c0daae09b9","reportUrl":"../103//#testresult/36b3d7c0daae09b9","status":"passed","time":{"start":1728265343574,"stop":1728265642037,"duration":298463}},{"uid":"4da6765b01cf8b6e","reportUrl":"../102//#testresult/4da6765b01cf8b6e","status":"passed","time":{"start":1728178877564,"stop":1728179185724,"duration":308160}},{"uid":"8c27db36a27202c6","reportUrl":"../101//#testresult/8c27db36a27202c6","status":"passed","time":{"start":1728092374516,"stop":1728092527235,"duration":152719}},{"uid":"6676d91387f30224","reportUrl":"../100//#testresult/6676d91387f30224","status":"passed","time":{"start":1728006066548,"stop":1728006226027,"duration":159479}},{"uid":"bf6e1b52708a064","reportUrl":"../99//#testresult/bf6e1b52708a064","status":"passed","time":{"start":1727919647452,"stop":1727919805906,"duration":158454}},{"uid":"5df20d4cca26df26","reportUrl":"../98//#testresult/5df20d4cca26df26","status":"passed","time":{"start":1727833249277,"stop":1727833420452,"duration":171175}},{"uid":"6c2c7b86e5366e69","reportUrl":"../97//#testresult/6c2c7b86e5366e69","status":"passed","time":{"start":1727746866183,"stop":1727747027412,"duration":161229}},{"uid":"6b1664daa73cefc4","reportUrl":"../96//#testresult/6b1664daa73cefc4","status":"passed","time":{"start":1727660432328,"stop":1727660584203,"duration":151875}},{"uid":"2032ba54d795689e","reportUrl":"../95//#testresult/2032ba54d795689e","status":"passed","time":{"start":1727574034144,"stop":1727574192081,"duration":157937}},{"uid":"f75690e86eb33347","reportUrl":"../94//#testresult/f75690e86eb33347","status":"passed","time":{"start":1727487588819,"stop":1727487756501,"duration":167682}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"b5d4c1757b82ac20.json","parameterValues":[]}